test/cans/staging/70184761-70fc-4b80-8c48-f406a478d5ab/2018-09-05T182535.846470Z/metadata.json

```diff
@@ -82,7 +82,11 @@
         "schema_type": "file",
         "file_core": {
             "file_name": "R1.fastq.gz",
-            "file_format": "fastq.gz"
+            "file_format": "fastq.gz",
+            "content_description": {
+                "text": "A text-based file containing both a biological sequence and its corresponding quality scores",
+                "ontology_label": "fastq"
```

Comment:
Let's use an actual value from the actual ontology. Study how ontologized fields are used in real-world bundles.

Reply:
Found a bundle to use on staging that has the field `content_description`